NewForce-Cohort-7/tabloidmvc-yellow-belly-yackers
# view current users posts

# Description
Completed ticket# 4 in Trello - [View Current User's Post](https://trello.com/c/LtsH95Lb)
Added a “My Posts” link to the header Menu (when logged in).
When the link is clicked, you are presented with a list of all the current posts by the user (in descending order)

## Type of change

# Testing Instructions
1. Clone down the repository.
2. git fetch -a
3. git checkout bh-currentUserPro
4. Navigate to the root directory for this repo /tabloidmvc-yellow-belly-yackers
5. Start TabloidMVC.sln
6. Make sure your db is created with seed data. Run [01_TabloidMVC_Create_DB.sql](https://github.com/NewForce-Cohort-7/tabloidmvc-yellow-belly-yackers/blob/main/SQL/01_TabloidMVC_Create_DB.sql) and [02_TabloidMVC_Seed_Data.sql](https://github.com/NewForce-Cohort-7/tabloidmvc-yellow-belly-yackers/blob/main/SQL/02_TabloidMVC_Seed_Data.sql)
8. Run it.
9. Log in as admin@example.com
10. Click MY POSTS at the top of the page.
11. You will be directed to a page of the current user’s posts with the most recent and only post on the list. Which is titled C# is the best language.
12. Click the “create new” button under the MYPOSTS header and you will be directed to a form for adding a new post.
13. Add a new post to test the functionality. Once you add the title, image url, choose a category and published date, click the "save" button. You will then be redirected to the details page of the new post you just created. 
14. Navigate back to the MYPOSTS tab on the navbar to see the new post you added on the list. 
15. You can also view previous posts by clicking the view post button. It’s right beside the pencil (edit button) and looks like an eyeball (o) but sideways. 
That will take you to the details of the selected post.


# Checklist:
